fcrepo-api-x-routing/README.md

```diff
@@ -0,0 +1,57 @@
+# API-X Routing module
+
+This module contains the API-X [routing](../src/site/markdown/execution-and-routing.md#routing) and [generic](../src/site/markdown/execution-and-routing.md#generic-endpoint-proxy) [implementations](../src/site/markdown/execution-and-routing.md#generic-intercepting-proxy) of execution components.
+
+## OSGi Deployment
+
+This module is automatically deployed when the `fcrepo-api-x` feature is installed in Karaf.
+
+## Configuration
```

Comment:
Some of them may also be cleaned up and deleted, as some sort of imply others too